mito-ai/mito_ai/completions/completion_handlers/agent_auto_error_fixup_handler.py

```diff
@@ -20,7 +20,8 @@ class AgentAutoErrorFixupHandler(CompletionHandler[AgentSmartDebugMetadata]):
     async def get_completion(
         metadata: AgentSmartDebugMetadata,
         provider: OpenAIProvider,
-        message_history: GlobalMessageHistory
+        message_history: GlobalMessageHistory,
+        model: str = None
```

Comment:
Based on the handlers.py code, I think this will never be None, so can we get rid of this default and have it be a required field?